applications/FluidDynamicsApplication/tests/adjoint_fluid_test.py

```diff
@@ -61,7 +38,7 @@ def _run_test(self,primal_parameter_file_name,adjoint_parameter_file_name):
             settings.AddValue("adjoint_settings", km.Parameters(adjoint_parameter_file.read()))
 
         # Add hdf5 output to the primal problem
-        settings["primal_settings"]["auxiliar_process_list"].Append(km.Parameters(r'''{
+        settings["primal_settings"]["processes"]["auxiliar_process_list"].Append(km.Parameters(r'''{
```

Comment:
because of this I think it is not added to a test-suite

Reply:
This test is already taken care of in #4117 (some of the added parameters need to be updated anyways).